Index: ../Mapa/app/src/main/res/values/styles.xml
===================================================================
--- ../Mapa/app/src/main/res/values/styles.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/res/values/styles.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,22 +0,0 @@
-<resources>
-
-    <!-- Base application theme. -->
-    <style name="AppTheme" parent="android:Theme.Holo.Light">
-        <!-- Customize your theme here.  Theme.AppCompat.Light.DarkActionBar -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
-    </style>
-
-    <style name="GenericProgressBackground" parent="android:Theme">
-        <item name="android:layout_width">fill_parent</item>
-        <item name="android:layout_height">fill_parent</item>
-        <item name="android:background">#DD111111</item>
-        <item name="android:gravity">center</item>
-    </style>
-    <style name="GenericProgressIndicator" parent="@android:style/Widget.ProgressBar.Small">
-        <item name="android:layout_width">wrap_content</item>
-        <item name="android:layout_height">wrap_content</item>
-        <item name="android:indeterminate">true</item>
-    </style>
-</resources>
Index: ../Mapa/app/src/main/res/values/colors.xml
===================================================================
--- ../Mapa/app/src/main/res/values/colors.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/res/values/colors.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
-    <color name="colorAccent">#FF4081</color>
-</resources>
Index: ../Mapa/.idea/runConfigurations.xml
===================================================================
--- ../Mapa/.idea/runConfigurations.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/.idea/runConfigurations.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="RunConfigurationProducerService">
-    <option name="ignoredProducers">
-      <set>
-        <option value="org.jetbrains.plugins.gradle.execution.test.runner.AllInPackageGradleConfigurationProducer" />
-        <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestClassGradleConfigurationProducer" />
-        <option value="org.jetbrains.plugins.gradle.execution.test.runner.TestMethodGradleConfigurationProducer" />
-      </set>
-    </option>
-  </component>
-</project>
\ No newline at end of file
Index: ../Mapa/.idea/vcs.xml
===================================================================
--- ../Mapa/.idea/vcs.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/.idea/vcs.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="VcsDirectoryMappings">
-    <mapping directory="$PROJECT_DIR$/../.." vcs="Git" />
-  </component>
-</project>
\ No newline at end of file
Index: ../Mapa/app/src/main/res/values/strings.xml
===================================================================
--- ../Mapa/app/src/main/res/values/strings.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/res/values/strings.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,4 +0,0 @@
-<resources>
-    <string name="app_name">Mapa</string>
-    <string name="title_activity_maps">Map</string>
-</resources>
Index: ../Mapa/app/build.gradle
===================================================================
--- ../Mapa/app/build.gradle	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/build.gradle	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,32 +0,0 @@
-apply plugin: 'com.android.application'
-
-android {
-    compileSdkVersion 22
-    buildToolsVersion '22.0.0'
-    defaultConfig {
-        applicationId "com.aeo.mapa"
-        minSdkVersion 16
-        targetSdkVersion 22
-        versionCode 1
-        versionName "1.0"
-    }
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
-    }
-    productFlavors {
-    }
-}
-
-dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:22.2.0'
-    compile 'com.google.android.gms:play-services:7.8.0'
-    compile 'com.google.android.gms:play-services-ads:7.8.0'
-    compile 'com.google.android.gms:play-services-identity:7.8.0'
-    compile 'com.google.android.gms:play-services-gcm:7.8.0'
-//    compile project(':library')
-    compile 'com.google.maps.android:android-maps-utils:0.4'
-}
Index: ../Mapa/app/src/main/res/layout/activity_maps.xml
===================================================================
--- ../Mapa/app/src/main/res/layout/activity_maps.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/res/layout/activity_maps.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,46 +0,0 @@
-<!--RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MapsActivity">
-
-    <fragment
-        android:id="@+id/map"
-        android:name="com.google.android.gms.maps.MapFragment"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_alignParentBottom="true"
-        android:layout_alignParentLeft="true"
-        android:layout_marginTop="44dp">
-
-
-</RelativeLayout-->
-
-<fragment xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:map="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/map"
-    android:name="com.google.android.gms.maps.SupportMapFragment"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context="com.aeo.mapa.MapsActivity">
-
-    <Button
-        android:id="@+id/btnSync"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="left|top"
-        android:layout_marginTop="20dp"
-        android:onClick="sync"
-        android:padding="10dp"
-        android:paddingRight="10dp"
-        android:text="Atualizar" />
-
-    <!--ProgressBar
-        android:id="@+id/progress_bar"
-        style="@android:style/Widget.ProgressBar.Small"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:indeterminate="true" /-->
-    <!--style="@style/GenericProgressIndicator"-->
-</fragment>
\ No newline at end of file
Index: ../Mapa/app/src/androidTest/java/com/aeo/mapa/ApplicationTest.java
===================================================================
--- ../Mapa/app/src/androidTest/java/com/aeo/mapa/ApplicationTest.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/androidTest/java/com/aeo/mapa/ApplicationTest.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,13 +0,0 @@
-package com.aeo.mapa;
-
-import android.app.Application;
-import android.test.ApplicationTestCase;
-
-/**
- * <a href="http://d.android.com/tools/testing/testing_android.html">Testing Fundamentals</a>
- */
-public class ApplicationTest extends ApplicationTestCase<Application> {
-    public ApplicationTest() {
-        super(Application.class);
-    }
-}
\ No newline at end of file
Index: ../Mapa/.gitignore
===================================================================
--- ../Mapa/.gitignore	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/.gitignore	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,9 +0,0 @@
-/app/build
-*.iml
-.gradle
-/local.properties
-/.idea/workspace.xml
-/.idea/libraries
-.DS_Store
-/build
-/captures
Index: ../Mapa/app/.gitignore
===================================================================
--- ../Mapa/app/.gitignore	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/.gitignore	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,1 +0,0 @@
-/build
Index: ../Mapa/app/src/main/AndroidManifest.xml
===================================================================
--- ../Mapa/app/src/main/AndroidManifest.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/AndroidManifest.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,42 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.aeo.mapa">
-
-    <!--
-         The ACCESS_COARSE/FINE_LOCATION permissions are not required to use
-         Google Maps Android API v2, but you must specify either coarse or fine
-         location permissions for the 'MyLocation' functionality. 
-    -->
-    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-
-    <application
-        android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
-        android:supportsRtl="true"
-        android:theme="@style/AppTheme">
-
-        <!--
-             The API key for Google Maps-based APIs is defined as a string resource.
-             (See the file "res/values/google_maps_api.xml").
-             Note that the API key is linked to the encryption key used to sign the APK.
-             You need a different API key for each encryption key, including the release key that is used to
-             sign the APK for publishing.
-             You can define the keys for the debug and release targets in src/debug/ and src/release/. 
-        -->
-        <meta-data
-            android:name="com.google.android.geo.API_KEY"
-            android:value="@string/google_maps_key" />
-
-        <activity
-            android:name=".MapsActivity"
-            android:label="@string/title_activity_maps">
-            <intent-filter>
-                <action android:name="android.intent.action.MAIN" />
-
-                <category android:name="android.intent.category.LAUNCHER" />
-            </intent-filter>
-        </activity>
-    </application>
-
-</manifest>
Index: ../Mapa/app/src/test/java/com/aeo/mapa/ExampleUnitTest.java
===================================================================
--- ../Mapa/app/src/test/java/com/aeo/mapa/ExampleUnitTest.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/test/java/com/aeo/mapa/ExampleUnitTest.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,15 +0,0 @@
-package com.aeo.mapa;
-
-import org.junit.Test;
-
-import static org.junit.Assert.*;
-
-/**
- * To work on unit tests, switch the Test Artifact in the Build Variants view.
- */
-public class ExampleUnitTest {
-    @Test
-    public void addition_isCorrect() throws Exception {
-        assertEquals(4, 2 + 2);
-    }
-}
\ No newline at end of file
Index: ../Mapa/.idea/misc.xml
===================================================================
--- ../Mapa/.idea/misc.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/.idea/misc.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,81 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="EntryPointsManager">
-    <entry_points version="2.0" />
-  </component>
-  <component name="NullableNotNullManager">
-    <option name="myDefaultNullable" value="android.support.annotation.Nullable" />
-    <option name="myDefaultNotNull" value="android.support.annotation.NonNull" />
-    <option name="myNullables">
-      <value>
-        <list size="4">
-          <item index="0" class="java.lang.String" itemvalue="org.jetbrains.annotations.Nullable" />
-          <item index="1" class="java.lang.String" itemvalue="javax.annotation.Nullable" />
-          <item index="2" class="java.lang.String" itemvalue="edu.umd.cs.findbugs.annotations.Nullable" />
-          <item index="3" class="java.lang.String" itemvalue="android.support.annotation.Nullable" />
-        </list>
-      </value>
-    </option>
-    <option name="myNotNulls">
-      <value>
-        <list size="4">
-          <item index="0" class="java.lang.String" itemvalue="org.jetbrains.annotations.NotNull" />
-          <item index="1" class="java.lang.String" itemvalue="javax.annotation.Nonnull" />
-          <item index="2" class="java.lang.String" itemvalue="edu.umd.cs.findbugs.annotations.NonNull" />
-          <item index="3" class="java.lang.String" itemvalue="android.support.annotation.NonNull" />
-        </list>
-      </value>
-    </option>
-  </component>
-  <component name="ProjectInspectionProfilesVisibleTreeState">
-    <entry key="Project Default">
-      <profile-state>
-        <expanded-state>
-          <State>
-            <id />
-          </State>
-          <State>
-            <id>Android Lint</id>
-          </State>
-        </expanded-state>
-        <selected-state>
-          <State>
-            <id>Abstraction issues</id>
-          </State>
-        </selected-state>
-      </profile-state>
-    </entry>
-  </component>
-  <component name="ProjectLevelVcsManager" settingsEditedManually="false">
-    <OptionsSetting value="true" id="Add" />
-    <OptionsSetting value="true" id="Remove" />
-    <OptionsSetting value="true" id="Checkout" />
-    <OptionsSetting value="true" id="Update" />
-    <OptionsSetting value="true" id="Status" />
-    <OptionsSetting value="true" id="Edit" />
-    <ConfirmationsSetting value="0" id="Add" />
-    <ConfirmationsSetting value="0" id="Remove" />
-  </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/build/classes" />
-  </component>
-  <component name="ProjectType">
-    <option name="id" value="Android" />
-  </component>
-  <component name="masterDetails">
-    <states>
-      <state key="ProjectJDKs.UI">
-        <settings>
-          <last-edited>1.8</last-edited>
-          <splitter-proportions>
-            <option name="proportions">
-              <list>
-                <option value="0.2" />
-              </list>
-            </option>
-          </splitter-proportions>
-        </settings>
-      </state>
-    </states>
-  </component>
-</project>
\ No newline at end of file
Index: ../Mapa/gradlew.bat
===================================================================
--- ../Mapa/gradlew.bat	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/gradlew.bat	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,90 +0,0 @@
-@if "%DEBUG%" == "" @echo off
-@rem ##########################################################################
-@rem
-@rem  Gradle startup script for Windows
-@rem
-@rem ##########################################################################
-
-@rem Set local scope for the variables with windows NT shell
-if "%OS%"=="Windows_NT" setlocal
-
-@rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=
-
-set DIRNAME=%~dp0
-if "%DIRNAME%" == "" set DIRNAME=.
-set APP_BASE_NAME=%~n0
-set APP_HOME=%DIRNAME%
-
-@rem Find java.exe
-if defined JAVA_HOME goto findJavaFromJavaHome
-
-set JAVA_EXE=java.exe
-%JAVA_EXE% -version >NUL 2>&1
-if "%ERRORLEVEL%" == "0" goto init
-
-echo.
-echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
-echo.
-echo Please set the JAVA_HOME variable in your environment to match the
-echo location of your Java installation.
-
-goto fail
-
-:findJavaFromJavaHome
-set JAVA_HOME=%JAVA_HOME:"=%
-set JAVA_EXE=%JAVA_HOME%/bin/java.exe
-
-if exist "%JAVA_EXE%" goto init
-
-echo.
-echo ERROR: JAVA_HOME is set to an invalid directory: %JAVA_HOME%
-echo.
-echo Please set the JAVA_HOME variable in your environment to match the
-echo location of your Java installation.
-
-goto fail
-
-:init
-@rem Get command-line arguments, handling Windowz variants
-
-if not "%OS%" == "Windows_NT" goto win9xME_args
-if "%@eval[2+2]" == "4" goto 4NT_args
-
-:win9xME_args
-@rem Slurp the command line arguments.
-set CMD_LINE_ARGS=
-set _SKIP=2
-
-:win9xME_args_slurp
-if "x%~1" == "x" goto execute
-
-set CMD_LINE_ARGS=%*
-goto execute
-
-:4NT_args
-@rem Get arguments from the 4NT Shell from JP Software
-set CMD_LINE_ARGS=%$
-
-:execute
-@rem Setup the command line
-
-set CLASSPATH=%APP_HOME%\gradle\wrapper\gradle-wrapper.jar
-
-@rem Execute Gradle
-"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %GRADLE_OPTS% "-Dorg.gradle.appname=%APP_BASE_NAME%" -classpath "%CLASSPATH%" org.gradle.wrapper.GradleWrapperMain %CMD_LINE_ARGS%
-
-:end
-@rem End local scope for the variables with windows NT shell
-if "%ERRORLEVEL%"=="0" goto mainEnd
-
-:fail
-rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
-rem the _cmd.exe /c_ return code!
-if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
-exit /b 1
-
-:mainEnd
-if "%OS%"=="Windows_NT" endlocal
-
-:omega
Index: ../Mapa/gradle.properties
===================================================================
--- ../Mapa/gradle.properties	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/gradle.properties	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,27 +0,0 @@
-# Project-wide Gradle settings.
-
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
-
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory settings.
-# Default value: -Xmx10248m -XX:MaxPermSize=256m
-# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
-
-org.gradle.parallel=true
-org.gradle.daemon=true
-org.gradle.jvmargs=-Xms256m -Xmx1024m
-
-#To fix compile error.
-sonatypeRepo=null
-sonatypeUsername=null
-sonatypePassword=null
\ No newline at end of file
Index: ../Mapa/settings.gradle
===================================================================
--- ../Mapa/settings.gradle	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/settings.gradle	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,1 +0,0 @@
-include ':app'
Index: ../Mapa/.idea/copyright/profiles_settings.xml
===================================================================
--- ../Mapa/.idea/copyright/profiles_settings.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/.idea/copyright/profiles_settings.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,3 +0,0 @@
-<component name="CopyrightManager">
-  <settings default="" />
-</component>
\ No newline at end of file
Index: ../Mapa/app/src/main/java/com/aeo/mapa/MapLoader.java
===================================================================
--- ../Mapa/app/src/main/java/com/aeo/mapa/MapLoader.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/java/com/aeo/mapa/MapLoader.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,168 +0,0 @@
-package com.aeo.mapa;
-
-
-import android.content.AsyncTaskLoader;
-import android.content.Context;
-import android.content.res.AssetManager;
-
-import com.aeo.mapa.model.Ocorrencia;
-import com.aeo.mapa.util.KmlReader;
-
-import org.xmlpull.v1.XmlPullParserException;
-
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.List;
-
-/**
- * Created by adriano on 30/11/2015.
- */
-public class MapLoader extends AsyncTaskLoader<InputStream> {
-
-    private InputStream mData;
-
-    public MapLoader(Context ctx) {
-        // Loaders may be used across multiple Activitys (assuming they aren't
-        // bound to the LoaderManager), so NEVER hold a reference to the context
-        // directly. Doing so will cause you to leak an entire Activity's context.
-        // The superclass constructor will store a reference to the Application
-        // Context instead, and can be retrieved with a call to getContext().
-        super(ctx);
-    }
-
-    @Override
-    public InputStream loadInBackground() {
-        InputStream is = null;
-
-        AssetManager am = getContext().getAssets();
-        try {
-            is = am.open("doc.kml");
-            KmlReader kmlReader = new KmlReader();
-            List<Ocorrencia> list = kmlReader.parse(is);
-            list.add(new Ocorrencia(null,null,null));
-        } catch (IOException e) {
-            e.printStackTrace();
-        } catch (XmlPullParserException e) {
-            e.printStackTrace();
-        }
-
-        return is;
-    }
-
-/********************************************************/
-    /** (2) Deliver the results to the registered listener **/
-    /********************************************************/
-
-    @Override
-    public void deliverResult(InputStream data) {
-        if (isReset()) {
-            // The Loader has been reset; ignore the result and invalidate the data.
-            releaseResources(data);
-            return;
-        }
-
-        // Hold a reference to the old data so it doesn't get garbage collected.
-        // We must protect it until the new data has been delivered.
-        InputStream oldData = mData;
-        mData = data;
-
-        if (isStarted()) {
-            // If the Loader is in a started state, deliver the results to the
-            // client. The superclass method does this for us.
-            super.deliverResult(data);
-        }
-
-        // Invalidate the old data as we don't need it any more.
-        if (oldData != null && oldData != data) {
-            releaseResources(oldData);
-        }
-    }
-
-    //*********************************************************//*
-    //** (3) Implement the Loader’s state-dependent behavior **//*
-    //*********************************************************//*
-
-    @Override
-    protected void onStartLoading() {
-        if (mData != null) {
-            // Deliver any previously loaded data immediately.
-            deliverResult(mData);
-        }
-
-        // Begin monitoring the underlying data source.
-//        if (mObserver == null) {
-//            mObserver = new SampleObserver();
-//            // TODO: register the observer
-//        }
-
-        if (takeContentChanged() || mData == null) {
-            // When the observer detects a change, it should call onContentChanged()
-            // on the Loader, which will cause the next call to takeContentChanged()
-            // to return true. If this is ever the case (or if the current data is
-            // null), we force a new load.
-            forceLoad();
-        }
-    }
-
-    @Override
-    protected void onStopLoading() {
-        // The Loader is in a stopped state, so we should attempt to cancel the
-        // current load (if there is one).
-        cancelLoad();
-
-        // Note that we leave the observer as is. Loaders in a stopped state
-        // should still monitor the data source for changes so that the Loader
-        // will know to force a new load if it is ever started again.
-    }
-
-    @Override
-    protected void onReset() {
-        // Ensure the loader has been stopped.
-        onStopLoading();
-
-        // At this point we can release the resources associated with 'mData'.
-        if (mData != null) {
-            releaseResources(mData);
-            mData = null;
-        }
-
-        // The Loader is being reset, so we should stop monitoring for changes.
-//        if (mObserver != null) {
-//            // TODO: unregister the observer
-//            mObserver = null;
-//        }
-    }
-
-    @Override
-    public void onCanceled(InputStream data) {
-        // Attempt to cancel the current asynchronous load.
-        super.onCanceled(data);
-
-        // The load has been canceled, so we should release the resources
-        // associated with 'data'.
-        releaseResources(data);
-    }
-
-    private void releaseResources(InputStream data) {
-        // For a simple List, there is nothing to do. For something like a Cursor, we
-        // would close it in this method. All resources associated with the Loader
-        // should be released here.
-    }
-
-    /*********************************************************************/
-    /** (4) Observer which receives notifications when the data changes **/
-    /*********************************************************************/
-
-    // NOTE: Implementing an observer is outside the scope of this post (this example
-    // uses a made-up "SampleObserver" to illustrate when/where the observer should
-    // be initialized).
-
-    // The observer could be anything so long as it is able to detect content changes
-    // and report them to the loader with a call to onContentChanged(). For example,
-    // if you were writing a Loader which loads a list of all installed applications
-    // on the device, the observer could be a BroadcastReceiver that listens for the
-    // ACTION_PACKAGE_ADDED intent, and calls onContentChanged() on the particular
-    // Loader whenever the receiver detects that a new application has been installed.
-    // Please don’t hesitate to leave a comment if you still find this confusing! :)
-//    private SampleObserver mObserver;
-}
Index: ../Mapa/app/src/release/res/values/google_maps_api.xml
===================================================================
--- ../Mapa/app/src/release/res/values/google_maps_api.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/release/res/values/google_maps_api.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,22 +0,0 @@
-<resources>
-    <!--
-    TODO: Before you release your application, you need a Google Maps API key.
-
-    To do this, you can either add your release key credentials to your existing
-    key, or create a new key.
-
-    Note that this file specifies the API key for the release build target.
-    If you have previously set up a key for the debug target with the debug signing certificate,
-    you will also need to set up a key for your release certificate.
-
-    Follow the directions here:
-
-    https://developers.google.com/maps/documentation/android/signup
-
-    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
-    string in this file.
-    -->
-    <string name="google_maps_key" templateMergeStrategy="preserve" translatable="false">
-        YOUR_KEY_HERE
-    </string>
-</resources>
Index: ../Mapa/app/src/main/java/com/aeo/mapa/util/KmlReader.java
===================================================================
--- ../Mapa/app/src/main/java/com/aeo/mapa/util/KmlReader.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/java/com/aeo/mapa/util/KmlReader.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,120 +0,0 @@
-package com.aeo.mapa.util;
-
-import android.util.Xml;
-
-import com.aeo.mapa.model.Ocorrencia;
-
-import org.xmlpull.v1.XmlPullParser;
-import org.xmlpull.v1.XmlPullParserException;
-
-import java.io.IOException;
-import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.List;
-
-/**
- * Created by adriano on 01/12/2015.
- */
-public class KmlReader {
-
-    private static final String PLACEMARK = "Placemark";
-    private static final String KML = "kml";
-    private static final String NAME = "name";
-    private static final String DESCRIPTION = "description";
-    private static final String COORDINATES = "coordinates";
-
-    private static final String ns = null;
-
-    public List<Ocorrencia> parse(InputStream in) throws XmlPullParserException, IOException {
-        try {
-            XmlPullParser parser = Xml.newPullParser();
-            parser.setFeature(XmlPullParser.FEATURE_PROCESS_NAMESPACES, false);
-            parser.setInput(in, null);
-            parser.nextTag();
-            return readKml(parser);
-        } finally {
-            in.close();
-        }
-    }
-
-    public List<Ocorrencia> readKml(XmlPullParser parser) throws IOException, XmlPullParserException {
-        List<Ocorrencia> list = new ArrayList<>();
-
-        parser.require(XmlPullParser.START_TAG, ns, KML);
-
-        while (parser.next() != XmlPullParser.END_TAG) {
-            if (parser.getEventType() != XmlPullParser.START_TAG) {
-                continue;
-            }
-            String name = parser.getName();
-            // Starts by looking for the entry tag
-            if (name.equals(PLACEMARK)) {
-                list.add(readEntry(parser));
-            } else {
-                skip(parser);
-            }
-        }
-
-        return list;
-    }
-
-    // Parses the contents of an entry. If it encounters a title, summary, or link tag, hands them off
-// to their respective "read" methods for processing. Otherwise, skips the tag.
-    private Ocorrencia readEntry(XmlPullParser parser) throws XmlPullParserException, IOException {
-        parser.require(XmlPullParser.START_TAG, ns, PLACEMARK);
-        String name = null;
-        String description = null;
-        String coordinates = null;
-        while (parser.next() != XmlPullParser.END_TAG) {
-            if (parser.getEventType() != XmlPullParser.START_TAG) {
-                continue;
-            }
-            String nameParser = parser.getName();
-            if (nameParser.equals(NAME)) {
-                name = readTag(parser, NAME);
-            } else if (nameParser.equals(DESCRIPTION)) {
-                description = readTag(parser, DESCRIPTION);
-            } else if (nameParser.equals(COORDINATES)) {
-                coordinates = readTag(parser, COORDINATES);
-            } else {
-                skip(parser);
-            }
-        }
-        return new Ocorrencia(name, description, coordinates);
-    }
-
-    private void skip(XmlPullParser parser) throws XmlPullParserException, IOException {
-        if (parser.getEventType() != XmlPullParser.START_TAG) {
-            throw new IllegalStateException();
-        }
-        int depth = 1;
-        while (depth != 0) {
-            switch (parser.next()) {
-                case XmlPullParser.END_TAG:
-                    depth--;
-                    break;
-                case XmlPullParser.START_TAG:
-                    depth++;
-                    break;
-            }
-        }
-    }
-
-    // Processes tags in the feed.
-    private String readTag(XmlPullParser parser, String tag) throws IOException, XmlPullParserException {
-        parser.require(XmlPullParser.START_TAG, ns, tag);
-        String title = readText(parser);
-        parser.require(XmlPullParser.END_TAG, ns, tag);
-        return title;
-    }
-
-    private String readText(XmlPullParser parser) throws IOException, XmlPullParserException {
-        String result = "";
-        if (parser.next() == XmlPullParser.TEXT) {
-            result = parser.getText();
-            parser.nextTag();
-        }
-        return result;
-    }
-
-}
Index: ../Mapa/build.gradle
===================================================================
--- ../Mapa/build.gradle	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/build.gradle	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,23 +0,0 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
-buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:1.5.0'
-
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
-    }
-}
-
-allprojects {
-    repositories {
-        jcenter()
-    }
-}
-
-task clean(type: Delete) {
-    delete rootProject.buildDir
-}
Index: ../Mapa/gradle/wrapper/gradle-wrapper.properties
===================================================================
--- ../Mapa/gradle/wrapper/gradle-wrapper.properties	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/gradle/wrapper/gradle-wrapper.properties	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,6 +0,0 @@
-#Wed Oct 21 11:34:03 PDT 2015
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-all.zip
Index: ../Mapa/.idea/compiler.xml
===================================================================
--- ../Mapa/.idea/compiler.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/.idea/compiler.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,22 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="CompilerConfiguration">
-    <option name="DEFAULT_COMPILER" value="Javac" />
-    <resourceExtensions />
-    <wildcardResourcePatterns>
-      <entry name="!?*.java" />
-      <entry name="!?*.form" />
-      <entry name="!?*.class" />
-      <entry name="!?*.groovy" />
-      <entry name="!?*.scala" />
-      <entry name="!?*.flex" />
-      <entry name="!?*.kt" />
-      <entry name="!?*.clj" />
-    </wildcardResourcePatterns>
-    <annotationProcessing>
-      <profile default="true" name="Default" enabled="false">
-        <processorPath useClasspath="true" />
-      </profile>
-    </annotationProcessing>
-  </component>
-</project>
\ No newline at end of file
Index: ../Mapa/app/proguard-rules.pro
===================================================================
--- ../Mapa/app/proguard-rules.pro	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/proguard-rules.pro	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,17 +0,0 @@
-# Add project specific ProGuard rules here.
-# By default, the flags in this file are appended to flags specified
-# in C:\AndroidSdk\sdk/tools/proguard/proguard-android.txt
-# You can edit the include path and order by changing the proguardFiles
-# directive in build.gradle.
-#
-# For more details, see
-#   http://developer.android.com/guide/developing/tools/proguard.html
-
-# Add any project specific keep options here:
-
-# If your project uses WebView with JS, uncomment the following
-# and specify the fully qualified class name to the JavaScript interface
-# class:
-#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
-#   public *;
-#}
Index: ../Mapa/.idea/modules.xml
===================================================================
--- ../Mapa/.idea/modules.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/.idea/modules.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/Mapa.iml" filepath="$PROJECT_DIR$/Mapa.iml" />
-      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
-    </modules>
-  </component>
-</project>
\ No newline at end of file
Index: ../Mapa/.idea/gradle.xml
===================================================================
--- ../Mapa/.idea/gradle.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/.idea/gradle.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="GradleSettings">
-    <option name="linkedExternalProjectsSettings">
-      <GradleProjectSettings>
-        <option name="distributionType" value="LOCAL" />
-        <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleHome" value="C:\Program Files\Android\Android Studio\gradle\gradle-2.8" />
-        <option name="gradleJvm" value="1.8" />
-        <option name="modules">
-          <set>
-            <option value="$PROJECT_DIR$" />
-            <option value="$PROJECT_DIR$/app" />
-          </set>
-        </option>
-      </GradleProjectSettings>
-    </option>
-  </component>
-</project>
\ No newline at end of file
Index: ../Mapa/app/src/main/java/com/aeo/mapa/MapsActivity.java
===================================================================
--- ../Mapa/app/src/main/java/com/aeo/mapa/MapsActivity.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/java/com/aeo/mapa/MapsActivity.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,275 +0,0 @@
-package com.aeo.mapa;
-
-import android.app.LoaderManager;
-import android.app.ProgressDialog;
-import android.content.Loader;
-import android.content.res.AssetManager;
-import android.os.Bundle;
-import android.support.v4.app.FragmentActivity;
-import android.view.Menu;
-import android.view.MenuInflater;
-import android.view.MenuItem;
-import android.view.View;
-
-import com.google.android.gms.maps.CameraUpdateFactory;
-import com.google.android.gms.maps.GoogleMap;
-import com.google.android.gms.maps.OnMapReadyCallback;
-import com.google.android.gms.maps.SupportMapFragment;
-import com.google.android.gms.maps.model.BitmapDescriptorFactory;
-import com.google.android.gms.maps.model.LatLng;
-import com.google.android.gms.maps.model.MarkerOptions;
-import com.google.maps.android.kml.KmlLayer;
-
-import org.xmlpull.v1.XmlPullParserException;
-
-import java.io.IOException;
-import java.io.InputStream;
-
-public class MapsActivity extends FragmentActivity implements OnMapReadyCallback,
-        LoaderManager.LoaderCallbacks<InputStream> {
-
-    private GoogleMap mMap;
-    private ProgressDialog progressDlg;
-    private KmlLayer kmlLayer;
-
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_maps);
-
-        // Obtain the SupportMapFragment and get notified when the map is ready to be used.
-        SupportMapFragment mapFragment = (SupportMapFragment) getSupportFragmentManager()
-                .findFragmentById(R.id.map);
-        mapFragment.getMapAsync(this);
-
-//        Handler mHandler = new Handler() {
-//            public void handleMessage(Message msg) {
-//                Bundle bundle = msg.getData();
-//                if (null != bundle){
-//                    int data = msg.getData().getInt("1");
-//                    if(msg.what == MSGCODE_SIZE){
-//                        mProgressBar.setMax(data);
-//                    } else if(msg.what == MSGCODE_PROGRESS){
-//                        mProgressBar.setProgress(data);
-//                    }
-//                }
-//            }
-//        };
-
-      }
-
-
-    /**
-     * Manipulates the map once available.
-     * This callback is triggered when the map is ready to be used.
-     * This is where we can add markers or lines, add listeners or move the camera. In this case,
-     * we just add a marker near Sydney, Australia.
-     * If Google Play services is not installed on the device, the user will be prompted to install
-     * it inside the SupportMapFragment. This method will only be triggered once the user has
-     * installed Google Play services and returned to the app.
-     */
-    @Override
-    public void onMapReady(GoogleMap googleMap) {
-        mMap = googleMap;
-
-        LatLng latLng = new LatLng(-22, -47);
-
-        mMap.addMarker(new MarkerOptions()
-                        .position(latLng)
-                        .title("Title")
-                        .snippet("Snippet")
-//                .icon(BitmapDescriptorFactory.fromResource(R.raw.img))
-                        .icon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_AZURE))
-//                .flat(true)
-        );
-
-        mMap.addMarker(new MarkerOptions()
-                        .position(new LatLng(-22.23, -47.23))
-                        .title("Title2")
-                        .snippet("Snippet2")
-                        .icon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_ORANGE))
-        );
-
-        mMap.moveCamera(CameraUpdateFactory.newLatLng(latLng));
-        mMap.setMapType(GoogleMap.MAP_TYPE_NORMAL);
-        mMap.animateCamera(CameraUpdateFactory.newLatLngZoom(latLng, 8.0f));
-        mMap.setMyLocationEnabled(true);
-        mMap.getUiSettings().setMyLocationButtonEnabled(true);
-        mMap.getUiSettings().setMapToolbarEnabled(true);
-        mMap.getUiSettings().setCompassEnabled(true);
-        mMap.getUiSettings().setZoomGesturesEnabled(true);
-    }
-
-    public void sync(View v) {
-
-        LoaderManager loadManager = getLoaderManager();
-
-        progressDlg = new ProgressDialog(this);
-        progressDlg.setMessage("Atualizando...");
-        progressDlg.setProgressStyle(ProgressDialog.STYLE_SPINNER);
-        progressDlg.setIndeterminate(true);
-        progressDlg.show();
-//        progressDlg = ProgressDialog.show(
-//                this, "Aguarde...", "Atualizando");
-
-        if (loadManager.getLoader(1) != null) {
-            loadManager.restartLoader(1, null, this);
-        } else {
-            loadManager.initLoader(1, null, this);
-        }
-//        MapTask mapTask = new MapTask();
-//        mapTask.execute();
-    }
-
-
-
-
-    @Override
-    public Loader<InputStream> onCreateLoader(int id, Bundle args) {
-        return new MapLoader(MapsActivity.this);
-    }
-
-    @Override
-    public void onLoadFinished(Loader<InputStream> loader, InputStream data) {
-        try {
-            if (kmlLayer != null) {
-                kmlLayer.removeLayerFromMap();
-            }
-            kmlLayer = new KmlLayer(mMap, data, getApplicationContext());
-            kmlLayer.addLayerToMap();
-            if (progressDlg != null && progressDlg.isShowing())
-                progressDlg.dismiss();
-
-        } catch (XmlPullParserException e) {
-            e.printStackTrace();
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-    }
-
-    @Override
-    public void onLoaderReset(Loader<InputStream> loader) {
-        kmlLayer.removeLayerFromMap();
-    }
-
-    /*
-    class MapTask extends AsyncTask<Void, Void, InputStream> {
-        ProgressDialog progressDlg;
-        @Override
-        protected InputStream doInBackground(Void... params) {
-            AssetManager am = getApplicationContext().getAssets();
-            InputStream is = null;
-            try {
-                is = am.open("Municipios_SP.kml");
-            } catch (IOException e) {
-                e.printStackTrace();
-            }
-
-            return is;
-        }
-
-        @Override
-        protected void onPreExecute() {
-            super.onPreExecute();
-            progressDlg = new ProgressDialog(getApplicationContext());
-            progressDlg.setMessage("Atualizando...");
-            progressDlg.setProgressStyle(ProgressDialog.STYLE_SPINNER);
-            progressDlg.setIndeterminate(true);
-            progressDlg.show();
-        }
-
-        @Override
-        protected void onPostExecute(InputStream is) {
-            super.onPostExecute(is);
-
-            try {
-                if (kmlLayer != null) {
-                    kmlLayer.removeLayerFromMap();
-                }
-                kmlLayer = new KmlLayer(mMap, is, getApplicationContext());
-                kmlLayer.addLayerToMap();
-                progressDlg.dismiss();
-
-            } catch (XmlPullParserException e) {
-                e.printStackTrace();
-            } catch (IOException e) {
-                e.printStackTrace();
-            }
-
-        }
-    }*/
-
-    private InputStream getStringFromAssetFile() {
-        AssetManager am = getAssets();
-        InputStream is = null;
-        try {
-            is = am.open("Municipios_SP.kml");
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-
-        return is;
-    }
-
-    private InputStream getStringFromAssetFile2() {
-        AssetManager am = getAssets();
-        InputStream is = null;
-        try {
-            is = am.open("doc.kml");
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-
-        return is;
-    }
-
-    @Override
-    public boolean onCreateOptionsMenu(Menu menu) {
-        MenuInflater inflater = getMenuInflater();
-        inflater.inflate(R.menu.main, menu);
-        return true;
-    }
-
-    @Override
-    public boolean onOptionsItemSelected(MenuItem item) {
-        super.onOptionsItemSelected(item);
-
-        int mapType = GoogleMap.MAP_TYPE_NORMAL;
-
-        switch (item.getItemId()) {
-            case R.id.normal_map:
-                mapType = GoogleMap.MAP_TYPE_NORMAL;
-                break;
-
-            case R.id.satellite_map:
-                mapType = GoogleMap.MAP_TYPE_SATELLITE;
-                break;
-
-            case R.id.terrain_map:
-                mapType = GoogleMap.MAP_TYPE_TERRAIN;
-                break;
-
-            case R.id.hybrid_map:
-                mapType = GoogleMap.MAP_TYPE_HYBRID;
-                break;
-        }
-
-        mMap.setMapType(mapType);
-        return true;
-    }
-
-    @Override
-    protected void onSaveInstanceState(Bundle outState) {
-        super.onSaveInstanceState(outState);
-//        outState.putString("isKml", isKml);
-    }
-
-    @Override
-    protected void onDestroy() {
-        super.onDestroy();
-        // Se estiver exibindo o progress, remova-o
-        if (progressDlg != null && progressDlg.isShowing())
-            progressDlg.dismiss();
-    }
-}
Index: ../Mapa/.idea/encodings.xml
===================================================================
--- ../Mapa/.idea/encodings.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/.idea/encodings.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="Encoding">
-    <file url="PROJECT" charset="UTF-8" />
-  </component>
-</project>
\ No newline at end of file
Index: ../Mapa/app/src/debug/res/values/google_maps_api.xml
===================================================================
--- ../Mapa/app/src/debug/res/values/google_maps_api.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/debug/res/values/google_maps_api.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,19 +0,0 @@
-<resources>
-    <!--
-    TODO: Before you run your application, you need a Google Maps API key.
-
-    To get one, follow this link, follow the directions and press "Create" at the end:
-
-    https://console.developers.google.com/flows/enableapi?apiid=maps_android_backend&keyType=CLIENT_SIDE_ANDROID&r=0E:49:E7:E1:C8:C9:FB:DC:58:0C:46:35:72:C7:09:1D:31:1C:0B:42%3Bcom.aeo.mapa
-
-    You can also add your credentials to an existing key, using this line:
-    0E:49:E7:E1:C8:C9:FB:DC:58:0C:46:35:72:C7:09:1D:31:1C:0B:42;com.aeo.mapa
-
-    Alternatively, follow the directions here:
-    https://developers.google.com/maps/documentation/android/start#get-key
-
-    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
-    string in this file.
-    -->
-    <string name="google_maps_key" templateMergeStrategy="preserve" translatable="false">AIzaSyAC_8_VntwgcVXYNkV4nu9ZsuI45OkAalk</string>
-</resources>
Index: ../Mapa/app/src/main/java/com/aeo/mapa/model/Ocorrencia.java
===================================================================
--- ../Mapa/app/src/main/java/com/aeo/mapa/model/Ocorrencia.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/java/com/aeo/mapa/model/Ocorrencia.java	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,18 +0,0 @@
-package com.aeo.mapa.model;
-
-/**
- * Created by adriano on 01/12/2015.
- */
-public class Ocorrencia {
-
-    private String name;
-    private String description;
-    private String coordinates;
-
-
-    public Ocorrencia(String name, String description, String coordinates) {
-        this.name = name;
-        this.description = description;
-        this.coordinates = coordinates;
-    }
-}
Index: ../Mapa/app/src/main/res/menu/main.xml
===================================================================
--- ../Mapa/app/src/main/res/menu/main.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
+++ ../Mapa/app/src/main/res/menu/main.xml	(revision b70b86a983bb58d7a8ec7d67848dbc907fa512b2)
@@ -1,25 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<menu xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:id="@+id/action_settings"
-        android:title="Tipo do Mapa">
-        <menu>
-            <item
-                android:id="@+id/normal_map"
-                android:title="Normal"/>
-            <item
-                android:id="@+id/satellite_map"
-                android:title="Satélite"/>
-            <item
-                android:id="@+id/terrain_map"
-                android:title="Terreno"/>
-            <item
-                android:id="@+id/hybrid_map"
-                android:title="Híbrido"/>
-        </menu>
-    </item>
-    <!--item android:id="@+id/menuUpdate"
-        android:icon="@drawable/ic_action_cancel"
-        android:orderInCategory="4"
-        android:showAsAction="always"
-        android:title="Atualizar"/-->
-</menu>
\ No newline at end of file
